src/py21cmfast/drivers/coeval.py

```diff
@@ -635,11 +624,12 @@ def _obtain_starting_point_for_scrolling(
     inputs: InputParameters,
     initial_conditions: InitialConditions,
     photon_nonconservation_data: dict,
-    cache: OutputCache,
-    regenerate: bool,
     minimum_node: int | None = None,
+    **iokw,
```

Comment:
Again, I prefer explicit arguments here too. The `**kw` arguments are useful when either:

1. You're immediately passing on a large subset of arguments to another function without much (preferably without any) amendment
2. The function parameters change a lot over (development) time, and there are many different viable combinations of parameters. In that case its sometimes easier to have a `**kw` and deal with the combinations via logic in the function. But this usually means you messed up the architecture earlier on.